src/dispatch.ts

```diff
@@ -115,84 +116,12 @@ function createHandler(seneca) {
     const taskResponse = await actAuthenticated(seneca, pattern)
 
     if (taskResponse.error || taskResponse.reject) {
-      log('queue error', pattern, taskResponse)
+      error('queue error', pattern, taskResponse)
     }
 
     return taskResponse
   }
 }
-
```

Comment:
These changes are just where I've moved the test sections to be next to the function they're testing